lib/gcpspanner/spanneradapters/backend_test.go

```diff
@@ -866,7 +877,7 @@ func CompareFeatures(f1, f2 backend.Feature) bool {
 		return false
 	}
 
-	if !compareImplementationStatus(f1.BrowserImplementations, f1.BrowserImplementations) {
+	if !compareImplementationStatus(f1.BrowserImplementations, f2.BrowserImplementations) {
```

Comment:
This is a small typo fix that is included here.

Reply:
Thanks for catching this!